### PR TITLE
Vinh notification

### PR DESCRIPTION
**Pull request by VinhH2402**
fixing bug on cancel route
add column 'confimed' to DB,
add function when rider confirm or cancel route will change value in confirmed column

### CODE REVIEW

File: db/models/routes.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const Route = db.define('route', {
   lngPickUp: Sequelize.DECIMAL(11,8),
   latDropOff: Sequelize.DECIMAL(10,8),
   lngDropOff: Sequelize.DECIMAL(11,8),
+  confirmed: Sequelize.BOOLEAN
 });
```
Comment (cmac0351) [inline db/models/routes.js:24]: should make this default value = false, right?

`confirmed: {
  type: Sequelize.BOOLEAN,
  defaultValue: false
}`

**Review by cmac0351 — COMMENTED** [review]
nice work on this notification feature.

File: server/controllers/riderControllers.js [hunk 2]
```diff
@@ -95,17 +97,16 @@ const confirmRoute = async (req, res) => {
 const cancelRoute = async (req, res) => {
   const socket = req.app.get('socket')
   try {
-    await Routes.update({rider_id: null}, {where: { id: req.body.id }})
+    await Routes.update({rider_id: null, confirmed: false}, {where: { id: req.body.id }})
```
Comment (sunikkim) [inline server/controllers/riderControllers.js:100]: nice work implementing ride confirmation!

**Review by sunikkim — APPROVED** [review]

File: client/src/components/Rider/Rider.jsx [hunk 3]
```diff
@@ -116,7 +115,7 @@ const Rider = (props) => {
 
   // needs to remove riderid from route
   const handleRideCancellation = () => {
-    const riderName = userData.first_name + userData.last_name;
+    const riderName = userData.first_name + ' ' + userData.last_name;
```
Comment (sunikkim) [inline client/src/components/Rider/Rider.jsx:118]: wonder if this should just be first name rather than the full name

**Review by cmac0351 — APPROVED** [review]

**Pull request merged**